server/test/repository/ExportServiceRepositoryTest.java

```diff
@@ -0,0 +1,137 @@
+package repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.google.common.collect.ImmutableList;
+import java.util.Locale;
+import java.util.NoSuchElementException;
+import models.LifecycleStage;
+import models.Question;
+import org.junit.Before;
+import org.junit.Test;
+import services.LocalizedStrings;
+import services.question.QuestionOption;
+import services.question.QuestionService;
+import services.question.types.MultiOptionQuestionDefinition;
+import services.question.types.QuestionDefinition;
+import services.question.types.QuestionDefinitionBuilder;
+import services.question.types.QuestionDefinitionConfig;
+
+public class ExportServiceRepositoryTest extends ResetPostgres {
```

Comment:
You've covered all the edge cases I can think of here, and several I hadn't though of, except one: Can we also add a test that makes sure removed options are still in the list?

Reply:
Sure thing.